wasmtime-environ/Cargo.toml

```diff
@@ -20,6 +20,7 @@ failure = { version = "0.1.3", default-features = false }
 failure_derive = { version = "0.1.3", default-features = false }
 indexmap = "1.0.2"
 rayon = "1.0"
+wasmtime-debug = { path = "../wasmtime-debug", default-features = false }
```

Comment:
It's unfortunate to add wasmtime-debug as a dependency here, because that pulls in gimli and faerie, which some users of wasmtime-environ may not otherwise need. Would it be acceptable, and sufficient, to move `ModuleAddressMap`, `FunctionAddressMap`, and `InstructionAddressMap` into wasmtime-environ?